public/app/plugins/panel/barchart/module.tsx

```diff
@@ -228,7 +233,13 @@ export const plugin = new PanelPlugin<PanelOptions, PanelFieldConfig>(BarChartPa
       description: 'Use the color value for a sibling field to color each bar value.',
     });
 
-    commonOptionsBuilder.addTooltipOptions(builder);
+    if (
+      !context.options?.fullHighlight ||
+      (context.options?.fullHighlight && context.options?.stacking === StackingMode.None)
```

Comment:
can this be simplified to `if (!context.options?.fullHighlight || context.options?.stacking === StackingMode.None)`

Reply:
> nice, lgtm.
> 
> maybe 2 minor points:
> 
> should we re-target to 9.4 since it's not breaking any more?
> 
> also, naming? maybe `Highlight full area on hover`?
> 
> ![image](https://user-images.githubusercontent.com/43234/211609712-7b85f3a4-9966-4824-b233-6599d0c2dc42.png)

Definitely should go for 9.4 now.

As for the naming I'd say: `Highlight full *bar* area on hover` What do you think?